app/src/main/java/com/github/dedis/student20_pop/model/Lao.java

```diff
@@ -0,0 +1,37 @@
+package com.github.dedis.student20_pop.model;
+
+/**
+ * Class modeling a Local Autonomous Organization (LAO)
+ */
+public class Lao {
+
+    // When launching a LAO the following values are empty (they are not needed for now):
+    // unique id, organization id, witness id list, member id list, event id list)
+    private String name;
+
+    /**
+     * Constructor for a LAO
+     *
+     * @param name  the name of the LAO
+     */
+    public Lao(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
We may want a unique identifier (id uint32) to be the "element" that identifies a particular LAO in case two LAOs would like to have the same name. If this is however, already planned for later, please disregard.

Reply:
Yes, I will work on that later!